.github/workflows/dependencies-checks.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         # We run update of the lock, before setting up the virtual environment, as dependencies checks are specifically
         # designed to test the latest internal/external dependencies to flag issues.
         run: |
-          make update-install
+          make update-install && poetry show
```

Comment:
Add to `main-checks` as well. That was only missing one as it isn't pertinent for post release one.